lib/miq_automation_engine/service_models/miq_ae_service_cloud_volume.rb

```diff
@@ -7,4 +7,9 @@ class MiqAeServiceCloudVolume < MiqAeServiceModelBase
     expose :attach_volume,          :override_return => nil
     expose :detach_volume,          :override_return => nil
   end
+
+  def self.create_volume_task(ems, options = {})
+    ext_management_system = ExtManagementSystem.find(ems)
+    CloudVolume.create_volume_queue('admin', ext_management_system, options)
```

Comment:
You shouldn't hard code the `'admin'` user here as that bypasses all RBAC and could allow an unprivileged user access to create cloud volumes (also bypassing user/tenant based quota).

Try using `@workspace.ae_user` instead

Reply:
I added the user parameter to the function.